src/app/header/components/TermButtons.tsx

```diff
@@ -0,0 +1,38 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
+import React, { useContext, useState } from 'react';
+
+import { TermContext } from '../../context/TermContext';
+import { getCurrentTerms, getReadableTerm } from '../../shared/utils/terms';
+
+export function TermButtons(): JSX.Element {
+  const [status, setStatus] = useState([true, false, false]);
+
+  const { setTerm } = useContext(TermContext);
+
+  const terms = getCurrentTerms();
+
+  const onClick = (event: React.MouseEvent<HTMLElement, MouseEvent>) => {
+    event.preventDefault();
+    const name = event.currentTarget.getAttribute('name');
+    let idx = -1;
+    if (name) {
+      setTerm(name);
+      idx = terms.indexOf(name);
+    }
+    const status = [false, false, false];
+    status[idx] = true;
+    setStatus(status);
+  };
```

Comment:
This should be in the if statement since if it isn't there's a chance it'll index negative.